File: app/src/main/java/com/marvelcomics/brito/marvelcomics/ui/fragment/ItemOffSetDecorationHorizontal.java
package com.marvelcomics.brito.marvelcomics.ui.fragment;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class ItemOffSetDecorationHorizontal extends RecyclerView.ItemDecoration {

    private int mItemOffset;

    public ItemOffSetDecorationHorizontal(int itemOffset) {
        mItemOffset = itemOffset;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent,
                               RecyclerView.State state) {
        super.getItemOffsets(outRect, view, parent, state);
        outRect.set(mItemOffset, 0, mItemOffset, 0);
    }
}

File: app/src/main/java/com/marvelcomics/brito/view/fragment/ItemOffSetDecorationHorizontal.kt
package com.marvelcomics.brito.marvelcomics.view.fragment

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView

class ItemOffSetDecorationHorizontal(private val mItemOffset: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(
        outRect: Rect,
        view: View,
        parent: RecyclerView,
        state: RecyclerView.State
    ) {
        super.getItemOffsets(outRect, view, parent, state)
        outRect[mItemOffset, 0, mItemOffset] = 0
    }
}